includes/pages/admin_questions.php

```diff
@@ -22,7 +21,7 @@ function admin_new_questions()
 
     if ($page != 'admin_questions') {
         if (auth()->can('admin_questions')) {
-            $new_messages = count(DB::select('SELECT `QID` FROM `Questions` WHERE `AID` IS NULL'));
+            $new_messages = Question::where('answer_user_id', null)->count();
```

Comment:
Using the magic method `Question::whereAnswerUserID(null)` would be much more clearer ;)